xforce_connector.py

```diff
@@ -146,12 +149,6 @@ def whois(self, param, action_id):
                 'Error running IBM X_Force WHOIS. Details:' + err.message
             )
 
-        if 'xforce_whois' not in whois_results:
-            return self.set_status_save_progress(
-                phantom.APP_ERROR,
-                'Error running IBM X_Force WHOIS. Details:' + err.message
```

Comment:
The err logic here is wrong, as the variable `err` should never be available here.